packages/nc-gui-v2/components/cell/Text.vue

```diff
@@ -1,100 +1,35 @@
 <script setup lang="ts">
-import { computed, inject, onMounted, ref } from '#imports'
+import { inject, onMounted, ref } from '#imports'
 
 interface Props {
   modelValue: any
 }
 
-const { modelValue: value } = defineProps<Props>()
+const props = defineProps<Props>()
 
-const emit = defineEmits(['update:modelValue'])
+const emits = defineEmits(['update:modelValue'])
 
 const editEnabled = inject<boolean>('editEnabled', false)
 
 const root = ref<HTMLInputElement>()
 
-const localState = computed({
-  get: () => value,
-  set: (val) => emit('update:modelValue', val),
-})
+const vModel = useVModel(props, 'modelValue', emits)
 
 onMounted(() => {
   root.value?.focus()
 })
 
-/* export default {
-  name: 'TextCell',
-  props: {
-    value: [String, Object, Number, Boolean, Array],
+watch(
```

Comment:
For focusing you can try following syntax suggested by @bcakmakoglu 

```
const onSetRef = (el: HTMLInputElement) => {
   el.focus()
}

...

<input :ref="onSetRef" />
```

Reply:
updated